cli/lib/kontena/main_command.rb

```diff
@@ -9,25 +9,31 @@ class Kontena::MainCommand < Kontena::Command
     exit 0
   end
 
+  banner Kontena.pastel.green("Getting started:"), false
+  banner ' - Create a Kontena Master (see "kontena plugin search" for a list of', false
+  banner '   provisioning plugins)', false
+  banner ' - Or log into an existing master, use: "kontena master login <master url>"', false
+  banner ' - Read more about Kontena at https://www.kontena.io/docs/', false
+
+  subcommand "master", "Kontena Master specific commands", load_subcommand('master_command')
   subcommand "cloud", "Kontena Cloud specific commands", load_subcommand('cloud_command')
-  subcommand "logout", "Logout from Kontena Masters or Kontena Cloud accounts", load_subcommand('logout_command')
+  subcommand "node", "Node specific commands", load_subcommand('node_command')
   subcommand "grid", "Grid specific commands", load_subcommand('grid_command')
-  subcommand "app", "App specific commands", load_subcommand('app_command')
   subcommand "stack", "Stack specific commands", load_subcommand('stack_command')
   subcommand "service", "Service specific commands", load_subcommand('service_command')
+  subcommand "container", "Container specific commands", load_subcommand('container_command')
   subcommand "vault", "Vault specific commands", load_subcommand('vault_command')
-  subcommand "certificate", "LE Certificate specific commands", load_subcommand('certificate_command')
-  subcommand "node", "Node specific commands", load_subcommand('node_command')
-  subcommand "master", "Master specific commands", load_subcommand('master_command')
   subcommand "vpn", "VPN specific commands", load_subcommand('vpn_command')
-  subcommand "registry", "Registry specific commands", load_subcommand('registry_command')
-  subcommand "container", "Container specific commands", load_subcommand('container_command')
   subcommand "etcd", "Etcd specific commands", load_subcommand('etcd_command')
+  subcommand "certificate", "LE Certificate specific commands", load_subcommand('certificate_command')
+  subcommand "registry", "Registry specific commands", load_subcommand('registry_command')
   subcommand "external-registry", "External registry specific commands", load_subcommand('external_registry_command')
+  subcommand "volume", "Volume specific commands [EXPERIMENTAL]", load_subcommand('volume_command')
+  subcommand "app", "App specific commands", load_subcommand('app_command')
+  subcommand "plugin", "Plugin specific commands", load_subcommand('plugin_command')
   subcommand "whoami", "Shows current logged in user", load_subcommand('whoami_command')
-  subcommand "plugin", "Plugin related commands", load_subcommand('plugin_command')
+  subcommand "logout", "Logout from Kontena Masters or Kontena Cloud accounts", load_subcommand('logout_command')
```

Comment:
IMO the deprecated `logout` command should either be removed, or preferrably just hidden from the `--help` output.

The `logout` description that shows up in the `--help` is misleading without any indication of it being a deprecated command that doesn't do anything.